low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -145,10 +145,10 @@ class SSPLBundleGenerate(object):
         logger.info("SSPL Support Bundle Location: %s" % self.localTempPath+file_name)
 
     def get_ipmi_sel_data(self):
-        response, _, return_code = self._run_command("ipmitool sel list")
+        response, error, return_code = self._run_command("ipmitool sel list")
         if return_code != 0:
-            logger.error(
-            "Required ipmitool is missing from the Node server to fetch sel data..!")
+            logger.warn(
+            f"Unable to fetch SEL list data with an error : {error}")
```

Comment:
First 'ipmitool' command should pre-constructed based on the config and not hardcoded, i.e detect ipmi config to check on BMC_INTERFACE in use from sspl config. Put this code under common 

if BMC_INTERFACE:
default: system -> indicates to use inband kcs channel
default:lan -> indicates to use ipmi over lan

If ipmitool is configured to use IPMI over LAN, then read BMC IP & credentials from global config(check node_hw sensor) ipmit command should look like 
>  ipmitool  -H $host -U user -P<password> sel list

If this fails, then actual reason can be logged using {error} returned by command.

else it would be plain
> ipmitool sel list

This would fail with following error, which doesnt give clear picture to everyone, who doesnt have enough knowledge.
>Could not open device at /dev/ipmi0 or /dev/ipmi/0 or /dev/ipmidev/0: No such file or directory

So to give clear picture, in case of KCS channel, we can detect VM env and avoid executing ipmitool command and fail like it does for enclosure, something with log like
> Detected VM environment, can't reach BMC over inband KCS channel, avoiding BMC data collection using IPMI.

Reply:
This comment brings in requirement to improve on original code beyond the scope of the original tkt. Create a new tkt and address it seperatly

Reply:
I will address the above comments separately with https://jts.seagate.com/browse/EOS-20685